.github/CODEOWNERS

```diff
@@ -1,2 +1 @@
-# Team Octopus is the default CODEOWNER unless you choose to take on the role
-*  @axonivy-market/team-octopus
+*  @axonivy-market/team-wawa
```

Comment:
I totally support that we take ownership of this product. 
2 things:
- I'd also be codeowner here, please include me also in the wildcard selector
- In the Market I've made bad experience with generic team-responsibilities. It easily happens that nobody will actually work on the PR, since no specific person is assigned. So I'd rather add directly you as codeowner. What do you think?